docs/CONFIG.md

```diff
@@ -4567,7 +4567,7 @@ Enabled = true
 [GasEstimator]
 Mode = 'FixedPrice'
 PriceDefault = '20 gwei'
-PriceMax = '100 micro'
+PriceMax = '100 gwei'
```

Comment:
what does this config do?

Reply:
This is used in gas estimation. I'm not sure why it was set so high but it was causing the TXM to fail on simulated backend since it would result in astronomically priced TXs that would go over the maximum balance possible on the simulated backend.